samtranslator/model/eventsources/push.py

```diff
@@ -126,13 +127,13 @@ def to_cloudformation(self, **kwargs):
         dlq_queue_arn = None
         if self.DeadLetterConfig is not None:
             EventBridgeRuleUtils.validate_dlq_config(self.logical_id, self.DeadLetterConfig)
-            dlq_queue_arn, dlq_resources = EventBridgeRuleUtils.get_dlq_queue_arn_and_resources(self, source_arn)
+            dlq_queue_arn, dlq_resources = EventBridgeRuleUtils.get_dlq_queue_arn_and_resources(
+                self, source_arn, passthrough_resource_attributes
+            )
             resources.extend(dlq_resources)
 
         events_rule.Targets = [self._construct_target(function, dlq_queue_arn)]
 
-        if CONDITION in function.resource_attributes:
```

Comment:
Does `CONDITION` appear anywhere else?

Reply:
passthrough_resource_attributes above already includes CONDITION, so it's duplicated